src/install/f5/windows/install-windows.mdx

```diff
@@ -0,0 +1,29 @@
+---
+componentType: default
+headingText: Configure the F5 integration
+---
+
+1. In the Integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`, create a copy of the sample configuration file by running:
+
+    ```shell
+    copy F5-config.yml.sample F5-config.yml
+    ```
+
+2. Edit the `F5-config.yml` file. Here's a basic configuration example:
+
+    ```yml
+    integrations:
+    - name: nri-f5
+        env:
+        HOSTNAME: localhost
+        PORT: 443
+        USERNAME: f5_user
+        PASSWORD: f5_password
+        interval: 15s
+        labels:
+        environment: production
+        inventory_source: config/f5
+    ```
+
+You can find all the config options at the bottom of this doc along with more complex config examples. 
```

Comment:
same comment, if you can link to the section, i'd link. otherwise we could also just give them the exact h2 title so they can ctrl+f